ProcessLib/BoundaryCondition/BHEBottomDirichletBoundaryCondition.cpp

```diff
@@ -0,0 +1,78 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "BHEBottomDirichletBoundaryCondition.h"
+
+#include <algorithm>
+#include <logog/include/logog.hpp>
+#include <vector>
+#include "ProcessLib/Utils/ProcessUtils.h"
+
+namespace ProcessLib
+{
+void BHEBottomDirichletBoundaryCondition::getEssentialBCValues(
+    const double /*t*/, GlobalVector const& x,
+    NumLib::IndexValueVector<GlobalIndexType>& bc_values) const
+{
+    SpatialPosition pos;
+
+    bc_values.ids.clear();
+    bc_values.values.clear();
+
+    bc_values.ids.resize(_bc_values.ids.size());
+    bc_values.values.resize(_bc_values.values.size());
+
+    const size_t n_nodes = _T_in_values.ids.size();
+    double tmp_T_out(300.0);
```

Comment:
Add a comment to the magic value or a `TODO (Haibing) This needs improvement such and such to get this and that.`.

Reply:
not needed. removed.